src/libs/API.js

```diff
@@ -217,7 +216,15 @@ function Authenticate(parameters) {
         'partnerUserSecret',
     ], parameters, commandName);
 
-    return Network.post(commandName, {
+    // Don't run more than one Authentication requests at the same time
```

Comment:
Yeah, maybe we can create a new issue for this as well? I'm not quite seeing the path for how this can happen or why we should be concerned about it - but guessing you would be able to document it and we can address it separately if it's concerning?